LocalPackages/NewTabPage/Tests/NewTabPageTests/Helpers/MessageHelper.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MessageHelper.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import NewTabPage
+import XCTest
+
+final class MessageHelper<MessageName: RawRepresentable> where MessageName.RawValue == String {
```

Comment:
This is another refactoring where I moved all `handleMessage*` test helper functions, previously copy-pasted between test case classes, into a single place.